html-css/.hintrc

```diff
@@ -0,0 +1,18 @@
+{
+  "connector": {
+    "name": "local",
+    "options": {
+      "pattern": ["**", "!.git/**", "!node_modules/**"]
+    }
+  },
+  "extends": ["development"],
+  "formatters": ["stylish"],
+  "hints": [
+    "button-type",
+    "disown-opener",
+    "html-checker",
+    "meta-charset-utf-8",
+    "meta-viewport",
+    "no-inline-styles:error"
+  ]
+}
```

Comment:
- [ ] Kindly add these [contents](https://github.com/microverseinc/linters-config/blob/master/html-css/.github/workflows/linters.yml) in your linters.yml file so your linters will work :+1: